css/templates/gallery/components/_landing-images.css

```diff
@@ -0,0 +1,581 @@
+  overflow: hidden;
+}
+
+.gallery-landing-wall-scroll::after {
+  width: 0px;
+  height: 100%;
+}
+
+@media (--breakpoint-xs) {
+  .gallery-landing-wall-scroll {
+    margin-bottom: -3px; /* remove spacing from display inline-block */
+  }
+}
+
+@keyframes gallery-image-card-hover {
+  0% {transform: translateY(-2vh) rotateZ(360deg); }
+  50% {transform: translateY(2vh) rotateZ(360deg); }
+  100% {transform: translateY(-2vh) rotateZ(360deg); }
+}
+
+@keyframes gallery-image-rotate {
+  0% {transform: translateY(-1vh) rotateZ(360deg); }
+  50% {transform: translateY(1vh) rotateZ(360deg); }
+  100% {transform: translateY(-1vh) rotateZ(360deg); }
+}
+
+.gallery-landing-image-wrapped {
+  width: 25vh;
+  position: absolute;
+  display: block;
+  transform: translateZ(0);
+  transform-origin: 0px 0px 0px;
+
+  animation-name: gallery-image-card-hover;
+  animation-duration: 20s;
+  animation-iteration-count: infinite;
+  animation-timing-function: ease-in-out;
+  animation-fill-mode: both;
+
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.amp-mode-mouse .gallery-landing-image-link {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: 2;
+}
+
+.gallery-landing-image-wrapped:nth-child(odd) {
+  animation-name: gallery-image-rotate;
+}
+
+.gallery-landing-image-wrapped amp-img {
+  object-fit: cover;
+  box-shadow: 1vh 6vh 5vh var(--color-shadow);
+  transition: transform 300ms;
+  pointer-events: none;
+}
+
+.gallery-landing-image-wrap:first-child amp-img { background: #d97b7f }
+.gallery-landing-image-wrap:nth-child(2) amp-img { background: #127eb1 }
+.gallery-landing-image-wrap:nth-child(3) amp-img { background: #e4bf0b }
+.gallery-landing-image-wrap:nth-child(4) amp-img { background: #9f3f7a }
+.gallery-landing-image-wrap:nth-child(5) amp-img { background: #5870ed }
+.gallery-landing-image-wrap:nth-child(6) amp-img { background: #fe95a6 }
+.gallery-landing-image-wrap:nth-child(7) amp-img { background: #a8a697 }
+.gallery-landing-image-wrap:nth-child(8) amp-img { background: #c8acbc }
+.gallery-landing-image-wrap:nth-child(9) amp-img { background: #ee9d57 }
+.gallery-landing-image-wrap:nth-child(10) amp-img { background: #ddd1d2 }
+.gallery-landing-image-wrap:nth-child(11) amp-img { background: #8dbbc0 }
+.gallery-landing-image-wrap:nth-child(12) amp-img { background: #9bd1ee }
+.gallery-landing-image-wrap:nth-child(13) amp-img { background: #b48b86 }
+.gallery-landing-image-wrap:nth-child(14) amp-img { background: #764b4e }
+.gallery-landing-image-wrap:nth-child(15) amp-img { background: #aaaab3 }
+.gallery-landing-image-wrap:nth-child(16) amp-img { background: #aeb197 }
+.gallery-landing-image-wrap:nth-child(17) amp-img { background: #370afa }
+.gallery-landing-image-wrap:last-child amp-img { background: #c1c1bf }
+
+.gallery-landing-image-wrapped:hover amp-img {
+  transform: translateY(-1vh) scale(1.05);
+  transition: transform 200ms ease-in-out;
+}
+
+.gallery-landing-image-wrap:first-child {
```

Comment:
can you move the background colors inside the bigger definitions, that should save some bytes.

Reply:
The background colours are targeting the `amp-img` within the `.gallery-landing-image-wrap` though, moving them to the bigger definitions would break functionality.